modules/formulize/admin/save/element_options_save.php

```diff
@@ -251,6 +251,31 @@
   }
 }
 
+// replace all existing blank radio button selections to the currently selected default
+if(isset($_POST['replaceblankswithdefault']) AND $_POST['replaceblankswithdefault']==1 AND $ele_type == "radio") {
+  $default_entry = (isset($_POST['changeuservalues']) AND $_POST['changeuservalues']==1) ? array_search(1, $newValues) : array_search(1, $element->getVar('ele_value'));
+  if($default_entry) {
+    $sql = "SELECT `entry_id`, `".$element->getVar('ele_handle')."` FROM ".$xoopsDB->prefix("formulize_".$formObject->getVar('form_handle'));
+    if(!$res = $xoopsDB->query($sql)) {
+      print "Error replacing blank values for the options in element $ele_id";
+    } else {
+      $update_array = array();
+      // find the blank entries
+      while($res2 = $xoopsDB->fetchRow($res)) {

```

Comment:
Instead of looping through all the records, the rows could be updated with an SQL query like this:
UPDATE table SET field = 'New Value' where field = "" or field is NULL